indico/modules/auth/controllers.py

```diff
@@ -301,8 +347,17 @@ def _create_user(self, form, handler, pending_user):
                 'Check <a href="{url}">your profile</a> for further details and settings.')
         flash(Markup(msg).format(url=url_for('users.user_profile')), 'success')
         db.session.flush()
+        self._notify_admins_if_necessary(user)
         return handler.redirect_success()
 
+    def _notify_admins_if_necessary(self, new_user):
+        info = HelperMaKaCInfo.getMaKaCInfoInstance()
+        if info.getNotifyAccountCreation():
+            admins = db.session.query(User).filter_by(is_admin=True).all()
```

Comment:
`User.find_all(is_admin=True)`

Also, send a single email to all of them. And I'd go for the separate email list setting I suggested above.